pkgs/games/sm64ex/coop.nix

```diff
@@ -7,13 +7,13 @@
 
 callPackage ./generic.nix {
   pname = "sm64ex-coop";
-  version = "0.pre+date=2022-08-05";
+  version = "0.pre+date=2023-02-22";
```

Comment:
The nixpkgs convention for unstable package versioning is `unstable-YYYY-MM-DD`, but I think keeping the currently used scheme is also fine?

Reply:
Will change to `unstable-YYYY-MM-DD` for both sm64ex PRs to follow nixpkgs convention.